cli/azd/pkg/infra/azure_resource_manager.go

```diff
@@ -75,7 +75,7 @@ func (rm *AzureResourceManager) GetDeploymentResourceOperations(ctx context.Cont
 
 // GetResourceGroupsForDeployment returns the names of all the resource groups from a subscription level deployment.
 func (rm *AzureResourceManager) GetResourceGroupsForDeployment(ctx context.Context, subscriptionId string, deploymentName string) ([]string, error) {
-	deployment, err := rm.azCli.GetSubscriptionDeployment(ctx, subscriptionId, deploymentName)
+	deployment, err := rm.azCli.GetResourceGroupDeployment(ctx, subscriptionId, deploymentName, deploymentName)
```

Comment:
Shouldn't this still be a subscription level deployment?

Reply:
yes, got it right after failed tests :)